README.md

```diff
@@ -40,46 +39,48 @@ Then from the cmd line run the following command:
 
 This will install the gem on your local environment.
 
-The next step is to configure the default rails logger to use the insightops logger.  
+The next step is to configure the default rails logger to use the InsightOps logger.  
+Ensure you add a `require` to load in the package:
 
+    require 'insight_ops.rb'
 
 In your environment configuration file ( for production : `config/environments/production.rb`), add the following:
 
-    Rails.logger = Le.new('LOG_TOKEN', 'REGION')
+    Rails.logger = InsightOps.new('LOG_TOKEN', 'REGION')
```

Comment:
@msmlatic-r7 @eshanley-r7 If we're renaming the class names and module away from LE, should we rename it to InsightOps or to R7Insight? Python library uses R7Insight. This is a breaking change that would require customers consuming this library to change references of LE to the new name, so it'd be good to minimise the amount of times we have to do this.

Reply:
yep, already agreed. Put R7Insight! Thank you!!